package.json

```
@@ -10,6 +10,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc --p ./tsconfig-build.json && vite build",
+    "prepare": "npm run build",
```

this `prepare` script is included in build so there is duplicity, we should add `release` script that has both build with publish. 

```suggestion
    "release": "npm run build && npm publish",
```